purchases/src/main/kotlin/com/revenuecat/purchases/CoroutinesExtensionsCommon.kt

```diff
@@ -69,3 +70,36 @@ suspend fun Purchases.awaitPurchase(purchaseParams: PurchaseParams): PurchaseRes
         )
     }
 }
+
+/**
+ * Gets the StoreProduct(s) for the given list of product ids of type [type], or for all types if no type is specified.
+ *
+ * Coroutine friendly version of [Purchases.getProducts].
+ *
+ * @param [productIds] List of productIds
+ * @param [type] A product type to filter by
+ *
+ * @warning This function is marked as [ExperimentalPreviewRevenueCatPurchasesAPI] and may change in the future.
+ * Only available in Kotlin.
+ *
+ * @throws [PurchasesException] with a [PurchasesError] if there's an error retrieving the offerings.
+ * @return The fetched list of [StoreProduct].
```

Comment:
I believe if it doesn't find some of the products, it will just return a partial list... I wonder if we should add that to the documentation... Then again, we don't do it for the other version, so probably not needed

Reply:
I will add it everywhere, I think it's good information